src/components/base-address/BaseAddress.vue

```diff
@@ -170,7 +170,7 @@ import {
   spaceRules
 } from '@/components/base-address/factories'
```

Comment:
This might be a problem. The path resolves to "src/components/base-address/factories", which is fine in this repo (ie, for Storybook's usage) but won't work when Base Address is imported in another UI.

Try changing this to `from './factories`.

Reply:
That worked great, thanks 👍

Reply:
Great. Resolving.